src/frontend/reducers/task_Reducer.js

```diff
@@ -0,0 +1,31 @@
+import { taskConstants } from "../constants";
+
+const task_Reducer = (state, action) => {
+	switch (action.type) {
+		case taskConstants.GET_TASK:
+			return {
+				...state,
+				tasks: action.payload,
+			};
+		case taskConstants.ADD_TASK:
+			return {
+				...state,
+				tasks: action.payload,
+			};
+		case taskConstants.EDIT_TASK:
+			return {
+				...state,
+				tasks: action.payload,
+			};
+		case taskConstants.DELETE_TASK:
+			return {
+				...state,
+				tasks: action.payload,
+			};
+
```

Comment:
This can re-written as :  

     switch(action.type):
            case taskConstants.ADD_TASK:
            case taskConstants.GET_TASK:
            case taskConstants.EDIT_TASK:
            cast taskConstants.DELETE_TASK:
                       return {...state , tasks : action.payload}
                       
 - To avoide reptition of code you use switch like this. 
 
 *Have a look at*: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

Reply:
thanks for Pointing this out i have implemented it